.github/workflows/discussion_ping.yml

```diff
@@ -17,11 +17,23 @@ jobs:
       discussion_title: ${{ github.event.discussion.title }}
```

Comment:
Very nice! An small detail is that the `slack_url` variable in jobs isn't needed anymore :)

Reply:
Thank you!
- removed all the unused env variables with commit - [81ac6d7](https://github.com/WordPress/openverse/pull/3577/commits/81ac6d79b41427731b1d8d55d6416938e1ec1517)